lib/shared/media_view.dart

```diff
@@ -162,7 +162,7 @@ class _MediaViewState extends State<MediaView> with SingleTickerProviderStateMix
         },
         pageBuilder: (BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) {
           return ImageViewer(
-            url: widget.postViewMedia.media.first.thumbnailUrl ?? widget.postViewMedia.media.first.originalUrl!,
+            url: widget.postViewMedia.media.first.imageUrl,
```

Comment:
Just as a quick note, this might bring back longer loading times for the image URL since we're prioritizing the full image now rather than the thumbnail. This might not be too big of a deal though considering we're deferring the image dimensions!

Reply:
Hmm, that's a good point. I'm not sure the best way to handle this, though, because people could be equally upset about the long load times as the fact that we weren't showing the full-resolution image. It's kind of a lose-lose. 😆

Reply:
Relay used to have and "HD" button in the image viewer which had to be pressed to load full-res images.
You could override this with a setting to always load the full-res image, if bandwidth and time weren't a concern.

Reply:
That's definitely a good idea for a future enhancement.

* By default, load the thumbnail; have a button for the user to load full-res
* Have an option to load full-res always, only on Wi-Fi, etc. (similar to video options)